### ISSUE TITLE
Need more instructions please

### ISSUE THREAD

**Issue by MervisDiamonds** [PanderMusubi/umik-1#2]
Hi I do not quite understand stand the directions in the readme and I would like a more ELI5 steps.  I would like to generate and extended calibration file for my umik-1.  Where do you I put my default calibration file, what commands do I run to generate the new calibration file, and where do I find the new calibration file?

**Comment by PanderMusubi** [comment]
Just migrated the repo, more instructions will follow soon there.
